csv-validator-cmd/src/test/resources/uk/gov/nationalarchives/csv/validator/cmd/integrityCheck/WO_95/tech_acq_metadata_v1_WO95Y14B000.csvs

```diff
@@ -1,5 +1,5 @@
 version 1.0
-@totalColumns 33
+@totalColumns 33 @integrityCheck("file_path","excludeFolder")
```

Comment:
Ah! I see now this is in the `integrityCheck` folder - I thought you were modifying existing tests. All fine.